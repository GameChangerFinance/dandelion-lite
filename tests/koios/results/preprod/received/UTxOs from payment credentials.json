[
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3439865,
    "block_time": 1746262702,
    "datum_hash": null,
    "epoch_no": 213,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "c453739095158ea4530d3d0bf72c1798392bee6b26a16c960c1ef063a0532d1e",
    "tx_index": 0,
    "value": "9999835423"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3436694,
    "block_time": 1746180970,
    "datum_hash": null,
    "epoch_no": 213,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "f8b3cea81b836a972bb375af83c0c65138bc6d3392b4653a67078c5bad09aa29",
    "tx_index": 0,
    "value": "10000000"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3430435,
    "block_time": 1746019303,
    "datum_hash": null,
    "epoch_no": 212,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "0a524740e091c2ff86cd9b2f906acc1742947a28047de0f80d9e59e3f8155999",
    "tx_index": 0,
    "value": "9800000"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3425066,
    "block_time": 1745877455,
    "datum_hash": null,
    "epoch_no": 212,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "725fa3fba9a57d0a22e78a192de248430566996e89501afc5def203a1242131c",
    "tx_index": 0,
    "value": "9800000"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3422952,
    "block_time": 1745823861,
    "datum_hash": null,
    "epoch_no": 212,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "31dba88af6b80d4e9eedf2f984c579680a1b30169af70ec3bc25be775fd46a08",
    "tx_index": 0,
    "value": "6966437"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3422924,
    "block_time": 1745823139,
    "datum_hash": null,
    "epoch_no": 212,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "4dc92610248e58a8d39ff6abe76d93ca380a3861c391ff934c61fbb552a8fbc9",
    "tx_index": 0,
    "value": "12516428"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3422544,
    "block_time": 1745812643,
    "datum_hash": null,
    "epoch_no": 212,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "fce09155759d9c08ea2692e877c4821293e0e43cdd0ade0c50734ba4f2eb961c",
    "tx_index": 0,
    "value": "1000000"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3420146,
    "block_time": 1745748863,
    "datum_hash": null,
    "epoch_no": 212,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "bf4d10e4441424e415d524b6572c9b235b7d71a0e948aac3270fa7b7bbb27b86",
    "tx_index": 0,
    "value": "9800000"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3413613,
    "block_time": 1745576463,
    "datum_hash": null,
    "epoch_no": 211,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "4749efff11d94ab1842bd6a074880cc9fb4def3e1a6c3b3e4b7bac30cdcda5f7",
    "tx_index": 0,
    "value": "9999835599"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [
      {
        "asset_name": "55534454544f4b454e",
        "decimals": 0,
        "fingerprint": "asset164sm46ur3n85xjrzyel4m7vk6hn7s9zad4j7mn",
        "policy_id": "9287925018a6d5ec98e4a8ede6db1f9653991f5a7f984f9560b9d739",
        "quantity": "2000000"
      }
    ],
    "block_height": 3413160,
    "block_time": 1745563994,
    "datum_hash": null,
    "epoch_no": 211,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "c1c4a38e165d148a70b3fd8ce8dd17c44b948277232bd1a1d259269208f1f2ce",
    "tx_index": 0,
    "value": "1245590"
  }
]
