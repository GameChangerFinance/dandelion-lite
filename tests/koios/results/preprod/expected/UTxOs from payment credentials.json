[
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3520039,
    "block_time": 1748343926,
    "datum_hash": null,
    "epoch_no": 218,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "f17011b5f3b7abebd565bdbe2d4531529a17a6b44c00b2fc7c242cccd6aebd5c",
    "tx_index": 0,
    "value": "9800000"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3506066,
    "block_time": 1747986326,
    "datum_hash": null,
    "epoch_no": 217,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "58f9258d5387bc943f385ea564899e5bb111832e02c68c26bb8784aa1f22820b",
    "tx_index": 0,
    "value": "500000000"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3504572,
    "block_time": 1747947193,
    "datum_hash": null,
    "epoch_no": 217,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "d766cd9c60f3307d779892745047036d3fe578588f63ada8a454a1e51141bbf7",
    "tx_index": 0,
    "value": "1000000"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3504466,
    "block_time": 1747944645,
    "datum_hash": null,
    "epoch_no": 217,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "36921ea49b500af1c4f03f8f93c57da7a0308f737dbc52800a8986ae14d85c66",
    "tx_index": 0,
    "value": "1000000"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [
      {
        "asset_name": "48504a54",
        "decimals": 0,
        "fingerprint": "asset1krtj8s02mkcznmefatfr7t5ltx4ewu0pj9mdfk",
        "policy_id": "38102c03682a1eb979a016dafa7cee481976128ea9f8a1e522c78187",
        "quantity": "234214412"
      }
    ],
    "block_height": 3498109,
    "block_time": 1747778045,
    "datum_hash": null,
    "epoch_no": 216,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "cb505a5b8e85fa8efe0202e921fa65a3047406f50cd3bdaf41f475d39e10585f",
    "tx_index": 0,
    "value": "1155080"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3497513,
    "block_time": 1747762594,
    "datum_hash": null,
    "epoch_no": 216,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "2abd3ead7769186a51a6467561bf622888b6ed353e48e197d34dcce1a92253d8",
    "tx_index": 0,
    "value": "9400000000"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3496878,
    "block_time": 1747746609,
    "datum_hash": null,
    "epoch_no": 216,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "9caab8ffc7618e77d9aaedc12df95e5464098995d27df862796a9a4dbc84c748",
    "tx_index": 0,
    "value": "10000000000"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3485460,
    "block_time": 1747447496,
    "datum_hash": null,
    "epoch_no": 216,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "95a71471248662d614cdf28603c766fc6697663b89ee98e8cf496e210ed33db6",
    "tx_index": 0,
    "value": "100000000"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3484667,
    "block_time": 1747426056,
    "datum_hash": null,
    "epoch_no": 216,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "4e934212048440cb9b50d7a9f9c1b9d4afb8f96e95f138a2cc483be0bb7ff493",
    "tx_index": 0,
    "value": "98000000"
  },
  {
    "address": "addr_test1vzpwq95z3xyum8vqndgdd9mdnmafh3djcxnc6jemlgdmswcve6tkw",
    "asset_list": [],
    "block_height": 3484599,
    "block_time": 1747423950,
    "datum_hash": null,
    "epoch_no": 216,
    "inline_datum": null,
    "is_spent": false,
    "payment_cred": "82e016828989cd9d809b50d6976d9efa9bc5b2c1a78d4b3bfa1bb83b",
    "reference_script": null,
    "stake_address": null,
    "tx_hash": "f99cf3ef9192893b5cf2cbbc15a26f6ce102e26468f2d2517369f63e437623c4",
    "tx_index": 0,
    "value": "19830979"
  }
]
